dev-python/protobuf-python/protobuf-python-3.19.4.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 2008-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit distutils-r1
```

Comment:
Does PEP517 work?

Reply:
No:
```
 *   DISTUTILS_ARGS are not supported in PEP-517 mode
```
Ebuild has `DISTUTILS_ARGS=(--cpp_implementation)`.
Passing `--cpp_implementation` is required to enable C++ implementation.
[setup.py](https://github.com/protocolbuffers/protobuf/blob/v3.19.4/python/setup.py#L188-L198) contains:
```
def get_option_from_sys_argv(option_str):
  if option_str in sys.argv:
    sys.argv.remove(option_str)
    return True
  return False


if __name__ == '__main__':
  ext_module_list = []
  warnings_as_errors = '--warnings_as_errors'
  if get_option_from_sys_argv('--cpp_implementation'):
```